server.js

```diff
@@ -13,17 +13,11 @@ const taxonomyRouterV1 = require('./api/controllers/routes/v1/taxonomy');
 const agencyRouterV1 = require('./api/controllers/routes/v1/agency');
 const locationRouterV1 = require('./api/controllers/routes/v1/location');
 
-const taxonomyRouterV2 = require('./api/controllers/routes/v2/taxonomy');
-const agencyRouterV2 = require('./api/controllers/routes/v2/agency');
-const locationRouterV2 = require('./api/controllers/routes/v2/location');
-
 app.use('/api/v1/taxonomy', taxonomyRouterV1);
 app.use('/api/v1/agency', agencyRouterV1);
 app.use('/api/v1/location', locationRouterV1);
 
-app.use('/api/v2/taxonomy', taxonomyRouterV2);
-app.use('/api/v2/agency', agencyRouterV2);
-app.use('/api/v2/location', locationRouterV2);
+app.use(require('./api/routes.js').getRoutes('v2', 'ckan'));
```

Comment:
this is cool, allowing us to pass version and platform as an argument to route.

Reply:
Thanks...this should make it easier to inject mock services for testing and change data services quickly for memcaching or 2.0 of SCOS.